core/classes/class-rest-api.php

```diff
@@ -327,6 +328,7 @@ function( $slug ) {
 					'featured_image_thumbnail' => get_the_post_thumbnail( $post_id, 'thumbnail' ),
 					'attendees'                => ( $event->attendee ) ? $event->attendee->attendees() : array(),
 					'current_user'             => ( $event->attendee && $event->attendee->get( get_current_user_id() ) ) ? $event->attendee->get( get_current_user_id() ) : '',
+					'venue'                    => $venue_information
```

Comment:
`'venue'                    => $event->get_venue_information(),`
only using once, so no need to set variable; add trailing comma per standards.